src/App.js

```diff
@@ -1,12 +1,24 @@
 import './App.css';
-import React from 'react';
-import Calculator from './components/Calculator';
+import {
+  BrowserRouter as Router, Routes,
+  Route,
+} from 'react-router-dom';
+import Books from './components/BookForm';
+import Category from './components/Category';
+import Navbar from './components/Navbar';
+import Form from './components/Form';
 
-const App = () => (
-  <>
-    <Calculator />
-
-  </>
-);
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/Category" element={<Category />} />
+        <Route path="/Form" element={<Form />} />
```

Comment:
- [ ] Your app should only have 2 `<Route>`s and not 3, one is for the BookList component and the other for the Categories component. kindly remove the Form `<Route>` 🚮